ts/sagas/wallet.ts

```diff
@@ -751,9 +751,18 @@ export function* watchWalletSaga(
   // here it used to check for BPD enabled
   const contentClient = ContentClient();
 
+  // watch for load abi request
```

Comment:
I think something went wrong with the last merge. Some calls are duplicated

Reply:
welp, that's embarassing 😅 
sorry about that, should be fixed in [db09fe3](https://github.com/pagopa/io-app/pull/5442/commits/db09fe3a6689af7af97a2eccd463e5876dcf0210), LMK!